src/runtime/tests/refiner-test.ts

```diff
@@ -70,7 +70,21 @@ describe('refiner', () => {
           };
           ref.validateData(data);
       }, `Unresolved field name 'num2' in the refinement expression.`);
-  });
+    });
+    it('Throws error when field name not found in query refinement', () => {
+      assert.throws(() => {
```

Comment:
I think the assert should be separate from the assume/ assign steps.

Reply:
Thanks, noticing this pointed out some fairly glaring indentation issues in this file.

I've tried to fix all tests using this pattern incorrectly. PTAL
Note: Some of the fixes resulted in reusing test names, I will have to rename these duplicates later as this is an anti-pattern. Ideally we'd have a linter that would catch test name overlaps.

Reply:
Good work! I added a few more comments about test hygiene. There's a lot of updating needed for this file. I think that it might be worthwhile to create a tech debt issue for these tests.